### DISCUSSION TITLE
About the     <VisibleTodoList /> component

### DISCUSSION THREAD

**Discussion by abelardolg** [reduxjs/redux#3974] [Help] [answered]
Hi there,

Inside App.js, the VisibleTodoList component is added.
When I inspected its file ('../containers/VisibleTodoList'), I don't see that this component is declared. 🤔

Where is this component declared, please? 

Best regards.

**Comment by markerikson** [comment, marked as answer]
Which specific file are you pointing to?

Note that the projects in the `/examples/` folder are outdated at this point and need to be rewritten.

See the code in [the new "Essentials" and "Fundamentals" tutorials](https://redux.js.org/tutorials/index) for our recommended patterns of how to write Redux logic.

**Comment by abelardolg** [comment]
Thanks @markerikson . 

If the projects stored in /examples/ are outdated, they should be marked like that or unreachables.

Thanks and happy days, better week.

**Comment by markerikson** [comment]
We've got an open issue and some PRs to start updating them already:

https://github.com/reduxjs/redux/issues/3917

https://github.com/reduxjs/redux/pull/3922

https://github.com/reduxjs/redux/pull/3919
